core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/CollectInformation.scala

```diff
@@ -192,6 +192,17 @@ class CollectInformation(apps: Seq[ApplicationInfo]) extends Logging {
     }
   }
 
+  def getSQLCleanAndAligned: Seq[SQLCleanAndAlignIdsProfileResult] = {
+    val allRows = apps.flatMap { app =>
+      app.cleanSQLIdsToAlign
```

Comment:
We can improve performance here by checking if the eventlog is Databricks application or DeltaLake enabled.
There is `ClusterTagPropHandler.isDatabricks` which is a member of the AppBase class.
We might need another property to be set while the tools is processing the Read/Write Nodes.